src/content-single/EventContentItem/components/EventMedia.js

```diff
@@ -1,37 +1,87 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
+import gql from 'graphql-tag';
+import { useQuery, useMutation } from '@apollo/react-hooks';
 import { get } from 'lodash';
 
 import { Media } from 'ui';
 
-const EventMedia = ({ title, coverImage, videos, liveStreamSource }) => (
-  <div className="mb-2 px-3">
-    <Media
-      imageUrl={get(coverImage, 'sources[0].uri', '')}
-      videoUrl={
-        !!liveStreamSource && liveStreamSource !== ''
-          ? liveStreamSource
-          : get(videos, '[0].sources[0].uri', '')
+import ADD_ATTENDANCE from '../../../mutations/addAttendance';
```

Comment:
FYI in the **web-app**, the imports/babel stuff is configured such that you can define paths easily relative to the root `src` dir like this:

```suggestion
import ADD_ATTENDANCE from 'mutations/addAttendance';
```